crates/ruff_linter/resources/test/fixtures/pylint/E1519.py

```diff
@@ -0,0 +1,20 @@
+from functools import singledispatch, singledispatchmethod
+
+
+@singledispatch
+def convert_position(position):
+    pass
+
+class Board:
+    @singledispatch  # [singledispatch-method]
+    @classmethod
+    def convert_position(cls, position):
+        pass
+
+    @singledispatch  # [singledispatch-method]
+    def move(self, position):
+        pass
```

Comment:
Is it possible to catch @funcname.register as the pylint rule does?

Even if not, it would be good to include those examples in the tests in case we can do it in the future.

Reply:
Probably a bit harder (but possible) to catch `@funcname.register`, but agree we should add tests for it with commentary.